x-pack/plugins/enterprise_search/server/utils/get_sync_jobs_queries.ts

```diff
@@ -30,7 +30,7 @@ export const getOrphanedJobsCountQuery = (ids: string[], isCrawler?: boolean) =>
         must: [
           {
             term: {
-              'connector.service_type': CRAWLER_SERVICE_TYPE,
+              service_type: CRAWLER_SERVICE_TYPE,
```

Comment:
I don't understand these changes--these queries are run against the .elastic-connectors-sync-jobs index, and the service type in those indices should be under `connector.service_type` not `service_type`

Reply:
oh no 🤦 
Sorry my macro messed up ..., reverting them back.